testing/scenario_app/ios/Scenarios/ScenariosUITests/UnobstructedPlatformViewTests.m

```diff
@@ -29,8 +29,8 @@ - (void)testNoOverlay {
 
   XCUIElement* platform_view = app.textViews[@"platform_view[0]"];
   XCTAssertTrue([platform_view waitForExistenceWithTimeout:1.0]);
-  XCTAssertEqual(platform_view.frame.origin.x, 25);
-  XCTAssertEqual(platform_view.frame.origin.y, 25);
+  XCTAssertEqual(platform_view.frame.origin.x, 0);
```

Comment:
Since the frame of the platform view changed, there are some changes in the frame of the platform views and overlays in this test.